### PR TITLE
CLDR-16627 remove FFFF and FEFF characters

### PR DESCRIPTION
**Pull request by DavidLRowe**
CLDR-16627

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by DavidLRowe** [comment]
For 71 files (the 70 files in the CLDR ticket plus oc_ES.xml), a "$" is removed from the end of a `parseLenient` element. (This is a reserved code for FFFF and should not be in this element.)

In addition four occurrences of FEFF (zero width no break space / byte order mark) were removed from the ends of four strings in am.xml.

**Comment by srl295** [comment]
> For 71 files (the 70 files in the CLDR ticket plus oc_ES.xml), a "$" is removed from the end of a `parseLenient` element. (This is a reserved code for FFFF and should not be in this element.)
> 
> In addition four occurrences of FEFF (zero width no break space / byte order mark) were removed from the ends of four strings in am.xml.

do you think they were meant to be `$` or perhaps `\$` (escaped dollarsign)?

**Review by macchiati — APPROVED** [review]
Looks good, thanks!

**Comment by macchiati** [comment]
I think they were meant to be $, but people didn't realize that needed
escaping. I think in each case there was an literal $ in the set anyway.

File: common/main/ab.xml
Comment (limck613) [inline common/main/ab.xml:1]: Understand Cc

**Review by limck613 — APPROVED** [review]

**Review by limck613 — APPROVED** [review]

**Pull request merged**